### ISSUE TITLE
Visualizing simulation

### ISSUE THREAD

**Issue by tom-bou** [MCZhi/DTPP#8]
Do you have any specific script you used for visualizing closed-loop simulations?

**Comment by MCZhi** [comment]
Hi, @tom-bou. We do not have a specific script for visualizing simulations. After running the test, a nuBoard (dashboard) will pop up. You can use nuBoard to visualize the simulation results.

**Comment by tom-bou** [comment]
> Hi, @tom-bou. We do not have a specific script for visualizing simulations. After running the test, a nuBoard (dashboard) will pop up. You can use nuBoard to visualize the simulation results.

Okey, thank you! Do you mean that you visualize through the msgpack files then?

**Comment by MCZhi** [comment]
Yes, you can use the nuBoard command line to load the msgpack files to visualize the simulation result.

**Comment by 942411526** [comment]
hi  how do you use the nuBoard command to visualize msgpack files?
